content/direct-deposit-management/release-notes/2025-02-05.md

```diff
@@ -0,0 +1,5 @@
+Dependents are no longer allowed to make updates to direct deposit information. This change is implemented to prevent potential data conflicts with downstream systems.
+
+The Direct Deposit API checks the ID of the requester to determine if they are a Veteran or dependent. If they are a dependent, or if Veteran/dependent status cannot be determined, the API generates a HTTP 422 error to notify the requester that they are not allowed to make updates.  
```

Comment:
```suggestion
- Dependents can no longer update direct deposit information to prevent data conflicts with downstream systems.
- The API checks the requester's ID to determine if they are a Veteran or dependent. If the requester is a dependent or their status is unclear, the API returns an HTTP 422 error notifying the requester that they aren't allowed to make updates.
- If a Veteran’s account is flagged for potential fraud by the VA source system, the API response includes a fraud alert in the header.
```